pages/about/jobs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: .Gov jobs
+permalink: /get-gov/jobs/
+layout: layouts/info-page
+sidenav: true
+excerpt: Work with the .gov team.
+tags: about
+eleventyNavigation:
+  key: about
+  order: 3
+  title: .Gov jobs
+---
+  
+We are a team of program managers, technologists, and analysts. We help U.S. government organizations gain public trust by making .gov a well-known, reliable, and secure space online.
+
+## Role title
+
+Description of role…
+
+Link to job on CISA
```

Comment:
Can leave as is, but could also wrap it in a place holder link (i.e. to `[link to job on CISA](#)`) if want to highlight that this is placeholder